UnitsNet.Tests/QuantityTests.cs

```diff
@@ -51,5 +51,28 @@ private static void AssertQuantityInfoRepresentsLength(QuantityInfo<LengthUnit>
             Assert.Equal(lengthUnitNames, quantityInfo.UnitNames);
 #pragma warning restore 618
         }
+
+        [Fact]
+        public void TestGenericMultiplication()
+        {
+            var length1 = Length.FromInches(2.0);
+            var length2 = Length.FromMeters(2.0);
+
+            var calculated = Multiply(length1, length2);
+            Assert.Equal(length1 * length2, calculated);
+        }
+
+        private IQuantity Multiply(IQuantity left, IQuantity right)
```

Comment:
Is this Multiply method supposed to be moved into an implementation somewhere? Also it seems work in progress still.